### ISSUE TITLE
Blacklist predicate filtering doesn't work properly on arrays 

### ISSUE THREAD

**Issue by rostislav-simonik** [edy/redux-persist-transform-filter#12]
initial:
```
state = { a : { b: [1,2,3]  }
path: a.b
predicate: () => true
```
expected: 
```state = { a: { b: [] } ```
currently:
```state = { a: { b: [undefined, undefined, undefined] }```

**Comment by edy** [comment]
hi, could you make a failing test for this issue?

**Comment by rostislav-simonik** [comment]
Pull request #13

**Comment by edy** [comment]
thank you, i just released 0.0.14 with the fix
